.github/workflows/tox.yml

```
@@ -169,11 +177,11 @@ jobs:
         python -m tox
       env:
         TOXENV: ansible29
-    - name: "Test with tox: ansibledevel"
+    - name: "Test with tox: ansible210"
```

I'm curious if
```suggestion
    - name: Test with tox: ${{ env.TOXENV }}
```
will render the var from the outer scope of the step-specific one...